Library/Homebrew/cmd/test-bot.rb

```diff
@@ -155,9 +154,12 @@ def run
           working_dir.cd { exec(*@command) }
         end
         write.close
-        while line = read.gets
-          puts line if verbose
-          @output += line
+        while buf = read.read(32)
+          if verbose
+            print buf
+            $stdout.flush
+          end
+          @output << buf
```

Comment:
Ah, I see. Would be good to indicate what the `32` is by splitting it out into a variable. Also, it seems like `32` is probably more bytes than we want if we want to print every dot.